arm-network/2016-09-01/swagger/examples/RouteFilterCreate.json

```diff
@@ -0,0 +1,99 @@
+{
+  "parameters": {
+    "routeFilterName": "filterName",
+    "resourceGroupName": "rg1",
+    "api-version": "2016-12-01",
+    "subscriptionId": "subid",
+    "parameters": {
+      "location": "West US",
+      "tags": {
+        "key1": "value1"
+      },
+      "properties": {
+        "rules": [
+          {
+            "name": "ruleName",
+            "properties": {
+              "access": "Allow",
+              "routeFilterRuleType": "Community",
+              "communities": [
+                "12076:5030",
+                "12076:5040"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsofot.Network/routeFilters/filterName",
+        "etag": "w/\\00000000-0000-0000-0000-000000000000\\",
```

Comment:
@deepakrajendranmsft - This will be a problem. Here Ding will have to update the Resource definition that routeFilter is having an allOf on. This model is coming from 2016-09-01/swagger/network.json . RouteFilter.json wants to move to 2016-12-01 api-version. We would not want a swagger spec to reference a definition from a different api-version as that would create a mess. So, if Ding copies the network.json spec in 2016-12-01 api-version will it be ok ?